components/layout/style_ext.rs

```diff
@@ -63,6 +63,15 @@ pub struct AxesOverflow {
     pub y: Overflow,
 }
 
+impl Default for AxesOverflow {
+    fn default() -> Self {
+        AxesOverflow {
+            x: Overflow::Visible,
+            y: Overflow::Visible,
+        }
+    }
+}
+
```

Comment:
```suggestion
impl AxesOverflow {
    fn visible() -> Self {
        AxesOverflow {
            x: Overflow::Visible,
            y: Overflow::Visible,
        }
    }
}

```
nit: the semantics of "default" isn't clear here.

Reply:
Add reference for using `visible` as default value. Is it okay?

Reply:
In that case, we should implement `Default` for `Overflow`. But we don't have that pattern in Stylo.

I'm inclined to having a little bit verbosity than inconsistency.